src/index.html

```diff
@@ -1,14 +1,65 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        class="logo"
+        alt="logo"
+        href="#home"
+      >
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Sumsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+               href="#"
+               class="nav__link"
+               data-qa="hover">
+               Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
               Laptops & Computers
            </a>
```
correct formatting (read the checklist) - pay attention to indentation